addon/-private/utils.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
```

Comment:
This isn't needed, see the changes made in https://github.com/spencer516/ember-functional-modifiers/commit/401489ee5734333e8fb147b07301187eb5ccd76a

Reply:
woo woo!
pushed!